common/src/main/java/com/gentics/mesh/core/data/node/Node.java

```diff
@@ -61,7 +62,7 @@
  * this node and to the created nodes in order to create a project data structure. Each node may be linked to one or more {@link NodeGraphFieldContainer}
  * vertices which contain the language specific data.
  */
-public interface Node extends MeshCoreVertex<NodeResponse, Node>, CreatorTrackingVertex, Taggable, ProjectElement {
+public interface Node extends MeshCoreVertex<NodeResponse, Node>, CreatorTrackingVertex, Taggable, ProjectElement, BucketableElement {
```

Comment:
Yes thats correct. However for the `NodeIndexHandler` I needed to change `#getElementClass()` to `Class<? extends BucketableElement> getElementClass()`. Due to this change it is required to handle assign the interface. IMHO this is the least intrusive change. Otherwise I would need to rework the whole IndexHandler mechanism.